spriint1/cv.tema.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cv Lorena</title>
+</head>
+<body>
+    <h1>Mudure Lorena Daniela</h1>
+    <img src="./poza.cv.jpg"alt="poza.cv">
+</body>
+    <p> <h3>Date Contact</h3>
+        <a href="lorenamudure216@gmail.com">lorenamudure216@gmail.com</a>
+        <a href="0748819008">0748819008</a></p>
+        <p>
+           <span>Adresa:Strada Bucuriei, nr 20, Zalau, Salaj</span>
+        </p>
+        <p>
+          <span>Sexul: F;</span>
+          <span>Data nasterii: 21.06.2002</span>
+        </p>
+    <p>
+        <h3>Despre Mine</h3>
+        <p>
+            Sunt o persoana ambitioasa, dornica de a invata lucruri noi si de a ma dezvolta atat pe plan personal cat si profesional.
+            Imi place sa lucrez in echipa si sa imi ating obiectivele propuse.
+        </p>
+        <p>
+            <h3>Experienta</h3>
+            <p>
+                <p>2022-prezent</p>
+                <p>Oprator Prelucrare Date</p>
+                <p>Lorem ipsum dolor sit amet, consectetur
+                    adipiscing elit, sed do eiusmod tempor
+                    incididunt ut labore et dolore magna aliqua.
+                    Enim sit amet venenatis urna cursus eget nunc.
+                    Lorem ipsum dolor sit amet, consectetur
+                    adipiscing elit, sed do eiusmod tempor
+                    incididunt ut labore et dolore magna
+                    Enim sit amet venenatis urna cursus eget nunc.</p>
+            </p>
+        </p>
+    </p>
+    <p>
+        <h3>Education</h3>
+    </p>
+    <div>
+        <p>2021-2025</p>
+        <p>Facultatea de Inginerie Electrica si Tehnologia Informatiei, Oradea</p>
+        <p>Specializare: Calculatoare</p>
+        <p>2017-2021</p>
+        <p>Colegiul Tehnic "Iuliu Maniu", Salaj</p>
+        <p>Profil: Economic</p>
+    </div>
+    <p>
+        <h3>Abilitati</h3>
+    </p>
+    <div>
+        <p>Limba Engleza</p>
+        <p>Microsoft Office</p>
+        <p>HTML</p>
+        <p>CSS</p>
+        <p>JavaScript</p>
+    <div>
+        <h3>Obiectiv</h3>
```

Comment:
```suggestion
    <h3>Obiectiv</h3>
```
(optional) identare: h3 e frate cu div-ul de mai sus, prin urmare se afla in aceeasi coloana.